src/xunit.console/CommandLine.cs

```diff
@@ -395,6 +396,71 @@ protected XunitProject Parse(Predicate<string> fileExists)
             return project;
         }
 
+        static void IncludeFromFile(XunitProject project, String path)
+        {
+            using (var input = File.OpenText(path))
+            {
+                string line;
+                int linenum = 0;
+                var spaceOrTab = " \t".ToCharArray();
```

Comment:
I'm less familiar with the files modeling in this project,
But isn't there any place for holding such static vars which will never change?
If not, at least move such thing to a static member var in this class, to be rused, rhather than generating such conversions on each iteration.
What do you think?
(I know it is minor, but maybe such accumulated improvements might bring better performance.
and asssume developper's environment is a Raspberry-Pi...)

Reply:
I considered that, but this runs during command line processing at startup; we would expect it to run once at most. Thus, it would actually be more expensive to compute this whether needed or not and save it in a static forever when no longer needed.